resources/public/api/conf/1.0/schemas/create-self-assessment-request.json

```diff
@@ -46,6 +46,12 @@
               "type": "number",
               "description": "The total income declared in the tax return",
               "example": "21233.33"
+            },
+            "trustsIncome": {
+              "id": "/properties/taxReturns/items/trustsIncome",
+              "type": "number",
+              "description": "The total trusts income declared in the tax return",
```

Comment:
i feel we misuse a bit "total" and need consistency between fields. Maybe we should rename the descriptions:
"The total income declared in the tax return"
"The employments income declared in the tax return"
"The self-employments profit declared in the tax return"
"The trusts income declared in the tax return"